Vagrantfile

```diff
@@ -618,6 +618,17 @@ SCRIPT
   config.vm.provider :hyperv do |v, override|
     override.vm.synced_folder "www/", "/srv/www", :owner => "www-data", :mount_options => []
     override.vm.synced_folder "log/", "/var/log", :owner => "vagrant", :mount_options => []
+
+    if use_db_share == true then
+      # Map the MySQL Data folders on to mounted folders so it isn't stored inside the VM
+      config.vm.synced_folder "database/data/", "/var/lib/mysql", create: true, owner: 112, group: 115, mount_options: []
```

Comment:
This bit's already handled in `develop`

Reply:
right, but i forgot to change to `override.vm.synced_folder` for hyper-v